README.md

```diff
@@ -37,6 +37,14 @@ docker run --rm --name greptimedb-dashboard \
 To build image by yourself, or use `docker-compose` to setup both greptimedb and
 dashboard, see [instructions here](docker/README.md).
 
+## Screenshot
+
+<p align="center">
+  <picture>
+    <img alt="Dashboard Screenshot" src="src/assets/images/screenshot.png" width="400px">
+  </picture>
+</p>
```

Comment:
1. do not put screenshot into source folder, just put it in root
2. it's difficult to see any detail from this fixed width image, just use default markdown image tag and let's see the result.